providers/feature_dism.rb

```diff
@@ -50,13 +50,17 @@ def delete_feature(_name)
 
 def installed?
   @installed ||= begin
+    install_ohai_plugin unless node['dism_features']
```

Comment:
Will the install_ohai_plugin also populate the node object? Otherwise the next line could still error looking for that element in node.